lib/cli/src/Cardano/CLI.hs

```diff
@@ -665,6 +666,33 @@ cmdVersion = command "version" $ info cmd $ mempty
     exec = do
         putStrLn (showVersion version)
         exitSuccess
+{-------------------------------------------------------------------------------
+                            Commands - 'stake-pool'
+-------------------------------------------------------------------------------}
+
+cmdStakePool
+    :: forall t. (DecodeAddress t, EncodeAddress t)
+    => Mod CommandFields (IO ())
+cmdStakePool = command "stake-pool" $ info (helper <*> cmds) $ mempty
```

Comment:
maybe we should name it `stake` instead of `stake-pool` ? :thinking:

Reply:
I think it should be `stake-pool` cause this is what we are essentially listing. Jormungandr's `stake` endpoint shows more than just a stake pools but the whole stake distribution, i.e. stakes in stake pools, dangling stake and unassigned stake.

Reply:
ok, thanks for feedback - it is `stake-pools` already so no need for change